components/style/properties/shorthand/position.mako.rs

```diff
@@ -269,7 +269,7 @@
             if self.align_content == self.justify_content {
                 self.align_content.to_css(dest)
             } else {
-                self.justify_content.to_css(dest)?;
+                self.align_content.to_css(dest)?;
```

Comment:
We can also write this like as follows if you prefer (no strong preference, just pointing it out):

```rust
self.align_content.to_css(dest)?;

if self.align_content != self.justify_content {
    dest.write_str(" ")?;
    self.justify_content.to_css(dest)?;
}

Ok(())
```